asteria/src/global_context.cpp

```diff
@@ -0,0 +1,23 @@
+// This file is part of Asteria.
+// Copyleft 2018, LH_Mouse. All wrongs reserved.
+
+#include "precompiled.hpp"
+#include "global_context.hpp"
+#include "utilities.hpp"
+
+namespace Asteria {
+
+Global_context::~Global_context()
```

Comment:
`= default` and `{}` are probably the same (they are not the first declarations and the destructors are non-trivial anyway) but `{}` takes you two more lines.

Reply:
Yes, as there is no difference, it is reasonable to prefer `{ }` to `= default` which is less universal, not to mention that destructors get implicit `noexcept(true)` (we do not make use of throwing anywhere).